plugins/stats_table_procedure.cc

```diff
@@ -110,18 +122,18 @@ save(const std::string & filename) const
 void StatsTable::
 serialize(ML::DB::Store_Writer & store) const
 {
-    int version = 1;
+    int version = 2;
```

Comment:
This format is different than other persisted model.  I suggest we add the name of the algorithm that created the file as well.